problem.py

```diff
@@ -161,28 +182,26 @@ def _step_detection_precision(step_list_true, step_list_pred):
     Returns:
         float -- precision, between 0.0 and 1.0
     """
-    _check_step_list(step_list_true)
     _check_step_list(step_list_pred)
 
     if len(step_list_pred) == 0:  # empty prediction
         return 0.0
 
     n_correctly_predicted = 0
     detected_index_set = set()  # set of index of detected true steps
```

Comment:
This set seems to be not used anywhere in this function no?

Reply:
Maybe you should add a check that the `step_true` has not already been used no?

Reply:
yes, I forgot to use it. I will add it now

Reply:
solved in d1e73021a2b584f44b2fbff2c05bbdeb6b39c312